### PR TITLE
Send confirmation request to all peers if representatives list is empty…

### PR DESCRIPTION
**Pull request by SergiySW**
… or observed representatives weight is less than online_weight_minimum

### CODE REVIEW

File: rai/node/node.cpp [hunk 1]
```diff
@@ -3582,7 +3583,7 @@ void rai::active_transactions::announce_votes ()
 						}
 					}
 				}
-				if (!reps->empty () && node.online_reps.online_stake () != node.config.online_weight_minimum.number ())
+				if (!reps->empty () && node.config.online_weight_minimum.number () < total_weight)
```
Comment (rkeene) [inline rai/node/node.cpp]: Is this test correct ?

Comment (SergiySW) [reply rai/node/node.cpp]: If total_weight > online_weight_minimum from config

Comment (rkeene) [reply rai/node/node.cpp]: Okay, that's easier to parse -- seems good !

Comment (SergiySW) [reply rai/node/node.cpp]: Yeah, perhaps I should replace it. Opposite comparison seems counterintuitive

**Pull request merged**